ISISNeutronMuon/MDANSE
# Beta release 2024

**Description of work**
Final changes before the beta release of MDANSE 2.

closes #580
closes #579
closes #577
closes #576 
closes #582
closes #492 

**Fixes**
1. Updated version numbers of MDANSE and MDANSE_GUI,
2. Splash screen text changed to 'beta release',
3. Correction to the job progress bar, to ensure that a finished job is 100% complete.
4. Initialised the primary axis field in the plotting GUI with the correct axis name.
5. Correction to the Grid.py file, allowing to plot curves with different primary axes.

**To test**
Run a DOS job on a trajectory. Check that it reached 100% when finished.
Plot the results of DOS and VACF at the same timer. Switch the plot to Grid. All selected curves should be visible.
